localstack-core/localstack/aws/handlers/validation.py

```diff
@@ -22,19 +23,23 @@
 
 LOG = logging.getLogger(__name__)
 
-
-# TODO: replace with from importlib.resources.files when https://github.com/python/importlib_resources/issues/311 is
-#   resolved. Import from a namespace package is broken when installing in editable mode.
+# todo: replace with importlib.resources, which currently seems to have problems with plux
```

Comment:
nit: It would be great if you could share a bit more details (maybe in the PR description) what it means that "plux has problems with it"?

Reply:
As I was trying to import `importlib.resources` or the backport (`importlib_resources`) in this module I was getting errors like `error importing module localstack.aws.handlers.proxy: MultiplexedPath only supports directories` when making the entrypoints. E.g., in [this build](https://github.com/localstack/localstack/actions/runs/10800929263/job/29959988036?pr=11497) (create community endpoints step). I did not try to dig further tbh. cc @thrau